### DISCUSSION TITLE
Finding 9-cheveron address of a gate in survival?

### DISCUSSION THREAD

**Discussion by TheMagneticDude** [Povstalec/StargateJourney#101] [Q&A]
I know in older versions of the mod when stargates are broken in survival a tooltip would display its unique 9-chevron address, allowing that specific gate to be dialed from anywhere but in the latest version this appears to have been removed.  Is there another way without using CC to access the 9-cheveron address?

**Comment by lukaskabc** [comment]
You can use item called PDA, right-click the gate, then 9-chevron address will be displayed as "ID".
Tooltip is displayed only on classic stargates by default, but you can turn it on for all gates with config (`always_display_stargate_id`).

**Reply by Povstalec** [reply]
Technically the tooltip is displayed on Stargates that have the "DisplayID" set to true in their NBT data (which means you can make them display their ID with commands as well), but yes, by default it gets set to true for Classic Stargates and any Stargates you upgrade from Stargates with "DisplayID" set to true

Also in future versions, "ID" will be called "9-Chevron Address" and "Address" will be called "Encoded Address" - just something to keep in mind for people anyone who might visit this place in the future
